suite-native/module-device-onboarding/src/screens/CreateOrRecoverCrossroadsScreen.tsx

```diff
@@ -29,8 +35,8 @@ export const CreateOrRecoverCrossroadsScreen = ({
     const { applyStyle } = useNativeStyles();
     const updateOnboardingAnalytics = useSetAtom(updateOnboardingAnalyticsAtom);
 
-    const isDeviceOnboardingRecoveryEnabled = useFeatureFlag(
-        FeatureFlag.IsDeviceOnboardingRecoveryEnabled,
+    const isDeviceOnboardingRecoveryEnabled = useSelector((state: MessageSystemRootState) =>
+        selectIsFeatureEnabled(state, Feature.deviceOnboardingMobileRecovery, true),
```

Comment:
nitpic: `true` is the default default value for selectIsFeatureEnabled, so there’s no need to define it explicitly if it’s static. But I see we use it that way in many places, so never mind.